forui/lib/src/widgets/date_picker/date_picker_style.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:flutter/foundation.dart';
+
+import 'package:forui/forui.dart';
+import 'package:meta/meta.dart';
+
+/// A date picker's style.
+class FDatePickerStyle with Diagnosticable {
```

Comment:
Is `labelStyle` missing?

Reply:
Good catch!

Reply:
Label style is already included in textfieldStyle.

Reply:
Hmmm, that's fair. I don't have a good solution but is it lowkey whack for the label style to be nested o.o